checkers/ruby/divide_by_zero.test.rb

```diff
@@ -0,0 +1,21 @@
+# <expect-error>
+a = 1 / 0
+
+# <expect-error>
+b = a / 0
+
+# safe
+d = a / 2
+
+# Safe
+puts a/2
+
+# <expect-error>
+puts a/0
+ 
+# <expect-error> c is being used as divisor
+c = 0
+
+puts a/c 
+
+d = a/c
```

Comment:
Removing line 21 (d = a/c) does not raise the issue in line 17, even though the issue still exists in line 19.